_config.yml

```diff
@@ -56,14 +56,14 @@ downloads:
       zip: 2ad4eaabfd92d627baffc6c971e4b8987b38c06baf42dc2fc2e05131095499e7
   stable_snapshot:
     url:
-      bz2: https://ftp.ruby-lang.org/pub/ruby/stable-snapshot.tar.bz2
```

Comment:
Yeah, OK, I added a commit reverting the changes to this file. This removes `3.` from the ticket description, so I'll update that too.